gravitee-apim-portal-webui-next/src/app/api-details/api-details.component.html

```diff
@@ -55,6 +55,17 @@
         i18n="@@apiDocumentation"
         >Documentation</a
       >
+      @if (this.currentUser()) {
```

Comment:
I believe that current user will always equal true. You will need to specify if it is empty or not.
Another option: add a signal in the current user service that is `isAuthenticated` that is true or false.

Reply:
I'm using `currentUser.isAuthenticated`

Reply:
Done